kpconv_torch/io/xyz.py

```diff
@@ -23,6 +23,7 @@ def read_xyz(filepath, xyz_only=False):
         colors = data[:, 3:6].astype(np.uint8)
     else:
         colors = colors.shape[0] > 0
+    labels = np.full_like(data[:, 0], 0).astype(np.int8)
     if data.shape[1] == 4:
```

Comment:
I did not understand why you have added this line. Isn't the np.squeeze enough ?

Reply:
We initialize `labels` only for 4- and 7-columns cases. The 6-columns case was not covered, then there was a undefined-variable bug at the end of the function.

Reply:
what does the 6-columns cas correspond to? For me, 4-columns is when you do not have colors, 7 is when you have it, but 6?

Reply:
You may want to read files without classification information!

And by the way, we have to avoid that type of error, which is not really informative to the user.

Reply:
ok, understood, it is initialized and then it serves or not